frontend/src/app/sessions/feedback/feedback-dialog/feedback-dialog.component.ts

```diff
@@ -102,6 +102,24 @@ export class FeedbackDialogComponent {
     return Object.values(FeedbackRating);
   }
 
+  get promptText() {
+    if (
+      this.data.sessions.some(
+        (session) =>
+          session.version.tool.config.supported_project_types.length === 1 &&
+          session.version.tool.config.supported_project_types[0] === 'training',
```

Comment:
A session is not necessarily a training session if the tool supports trainings. Instead, check if the provisioned project is a training.

Also, usually a training contains of more than one session. Therefore, the case is usually not applied.